test/end-to-end/patient/registry.search.js

```diff
@@ -102,6 +103,13 @@ function PatientRegistrySearch() {
     expectNumberOfGridRows(NUM_MATCHING);
   });
 
+  it(`should find patients with origin location "${parameters.originVillageName}" `, () => {
```

Comment:
Nice test!

For APIs, it is easier to test them by writing an integration test.  They are both faster to execute and easier to debug.  The tests for the patients is in [this file](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/integration/patients.js).